src/quo/components/profile/expanded_collectible/view.cljs

```diff
@@ -0,0 +1,54 @@
+(ns quo.components.profile.expanded-collectible.view
+  (:require
+    [promesa.core :as p]
+    [quo.components.counter.collectible-counter.view :as collectible-counter]
+    [quo.components.profile.expanded-collectible.style :as style]
+    [quo.foundations.shadows :as shadows]
+    [quo.theme]
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(defn view-internal
+  [{:keys [container-style square? on-press counter image-src]}]
+  (let [[image-size set-image-size] (rn/use-state {})]
+    (rn/use-effect
+     (fn []
+       (p/let [[image-width image-height] (rn/image-get-size image-src)]
```

Comment:
getting the image to match the designs is tricky because we need to have the correct dimension of the image. 
By getting the image dimensions we can set the appropriate aspect ratio and from there create a container that perfectly wraps the image, and with that place the counter in a consistent position relative to the corner of the image regardless of it's size.

Reply:
Shouldn't we not be using `use-effect` anymore? This one seems like a valid use case, but maybe also check with @flexsurfer

Reply:
Was getting some weird errors trying that because that method is async.
@clauxx any ideas about this?

Reply:
oh it's `p/let` didn't notice that, can we do it more visible?

Reply:
@flexsurfer if we put it inside let, it will be called on every re-render, no?

Reply:
yes i didn't notice the promise, seems like use-effect is legit here

Reply:
Yeah the image source can change in theory.

Reply:
> can we do it more visible?

I guess replacing `p/let` with `p/then`/`.then` to make it clearer it's a promise could help

Reply:
imo, just changing the alias to promesa.

e.g `promesa/let`

Reply:
no strong preference from my side

Reply:
ok for the time being I would prefer that we alias as `promesa`. Hopefully it will be clearer then for other developers who are unfamiliar with the library/ to look out for this.

I think using `/let` is more succinct and it would be good to have some different examples of using promesa in the codebase for other developers to work from.

Reply:
going to do this in a separate pr to avoid the noise, has to be made across all the repo 👍